### ISSUE TITLE
vfs: Show empty folders as hydrated

### ISSUE THREAD

**Issue by johkoenig** [nextcloud/desktop#3251]
By default, empty folders are shown as cloud-saved only:
![Unbenannt](https://user-images.githubusercontent.com/42836313/116795883-6e974f80-aad8-11eb-88ad-27a6111b0807.png)

For me, this is misleading, because the cloud-symbol tells that inside this folder there is data which is not available on this PC. However all data is downloaded already (the empty folder exists on my PC), so I think a green "downloaded" mark would be much better.
In other words: no data -> no missing data -> make it green!

**Comment by allexzander** [comment]
@johkoenig 
Misleading indeed. But, it behaves similarly in OneDrive which is using the same API. We'll see if we can improve, though. Great point. Right now, I guess, you can make those folders "Always available on this device" via the context menu, just to make that green mark appear. As a workaround.

**Comment by johkoenig** [comment]
I just checked on my office PC where I use OneDrive. You're right, empty folders also show as "cloud". 
However, I think the better approach would be to use the cloud as symbol as "there is data which is not downloaded inside this folder". So, an empty folder should be light green by default and the moment somebody else places a file in it then changes to the cloud symbol.

**Comment by allexzander** [comment]
@johkoenig A great suggestion indeed. We'll see if we have some spare time to implement it somewhat later.

**Comment by Rello** [comment]
The client has improved a lot since the ticket was opened.
If it is still relevant, please create a fresh ticket.
Thank you for your support
